BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-143:Super Admin Profile

- Add profile layout to super admin
- Add reset, change password and save buttons

QA:
Hot to test?:
Log in as a Super Admin (email: oclacep@wikimedia.org, password: prueba1234)
When logged you should se a nav bar (temporary), move to the profile link.
In the profile you should see all inputs with the data of the current logged user.
The button "Change password" should open a modal with two passsword inputs, to try it, write the new password on both inputs (once you ended with the tests please change the password again to "prueba1234")
To try the "Save changes" button change the password that is in the password input.
The "reset" button should reset the fields you changed.
Notes:
The nav bar is temporaryand just to use on the test of the PRs, any bad function on navigation should be ignored.
The input "Password" will be deleted, but is necessary now to test the submit button.
